test/addons/openssl-client-cert-engine/test.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+const common = require('../../common');
+const assert = require('assert');
```

Comment:
A `hasCrypto` check should be added (before require('https')) to enable it to pass when `--without-ssl` is specified:
```javascript
if (!common.hasCrypto)
  common.skip('missing crypto');
```

Reply:
Fixed, thanks!